modules/apps/accessibility/accessibility-menu-web/npmscripts.config.js

```diff
@@ -1,19 +0,0 @@
-/**
- * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
- * This library is free software; you can redistribute it and/or modify it under
- * the terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your option)
- * any later version.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
- * details.
- */
-
-module.exports = {
-	build: {
-		main: 'src/main/resources/META-INF/resources/js/index.ts',
```

Comment:
Hey @matuzalemsteles Based on [Bryce's comment ](https://github.com/liferay-peds/liferay-portal/pull/123#issue-1769018083)won't removing this cause it to be loaded as an AMD module instead of as an ESM module, which will make it unavailable to frontend-js-react-web. As a result this wouldn't work, right?

Reply:
This is somewhat related to my previous comment, due to the changes to avoid the circular dependency now the `accessibility-menu-web` module no longer needs to be ESM the same for cookies because now we are no longer exposing the constants and the atom. I had to move to the `frontend-js-react-web` module to avoid this.

https://github.com/liferay-peds/liferay-portal/pull/122#issuecomment-1615225431

Reply:
Okay, I see how these are related. I don't think this is the way we want to fix the circular dependency since it actually ends up putting code that should be in the accessibility menu module in frontend-js-react-web. 

@matuzalemsteles Wouldn't another way of handling the circular dependency issue be to create a new module that exports our atom that both accessibility-menu-web and frontend-js-react-web could depend on? It could be something like accessibility-settings-state-web.

Reply:
> Wouldn't another way of handling the circular dependency issue be to create a new module that exports our atom that both accessibility-menu-web and frontend-js-react-web could depend on? It could be something like accessibility-settings-state-web.

That would be another solution as well, if that's something we want we can do that.

Reply:
Yeah, it's not ideal. But in general it seems Liferay is happier with having more small modules rather than adding unrelated code to another module. For instance what I want to avoid is in the future having to make a change to frontend-js-react-web when I'm adding a new accessibility menu setting.

If you're unsure about this, I'm happy to wait for Bryce to get back next week and we can ask him, or maybe someone else in frontend-infra could weigh in on this?

Reply:
I think we could go with this for now and create a ticket to make a better decision on this, whether we create a new module or move this elsewhere with help from Bryce. What do you think? I wanted to unblock this pull request as soon as possible to avoid being blocked by this, I think this is something we can change at any time.

Reply:
I definitely really want to get this unblocked as well, but I don't think we want to merge it like this. Notice for instance, we are importing CONSTANTS from frontend-js-react-web, but instead of those constants being related to the react, they are constants for the accessibility menu. 

I think it's worth the time to remove this coupling first.

Reply:
Yeah, so I think the best solution here would really be to create the module as suggested and we can discuss it later if this really is the best solution or if there are others with Bryce.

Reply:
Yeah, let's do that. Create a module to export the atom, get this merged and then discuss the best solution when Bryce is back.

Thanks.